src/portable/portable.c

```diff
@@ -264,16 +276,29 @@ static int extract_now(
                         }
 
                         if (socket_fd >= 0) {
+#if HAVE_SELINUX
+                                /* The units will be copied on the host's filesystem, so if they had a SELinux label
+                                 * we have to preserve it. Copy it out so that it can be applied later. */
+                                _cleanup_freecon_ char *con = NULL;
+
+                                r = fgetfilecon(fd, &con);
```

Comment:
hmm, we switched go "_raw" labels pretty much everywehere else i think, i.e. the calls that don't do weird IPC behind our back. I have the suspicion we should stick to that, here too?

Reply:
mh I thought I was already using the _raw